addons/mrp_subcontracting/tests/test_subcontracting_portal_ui.py

```diff
@@ -11,7 +11,7 @@ class TestSubcontractingPortalUi(HttpCase):
     def setUpClass(cls):
         super().setUpClass()
         # 1. Create portal user
-        user = cls.env['res.users'].with_context({'no_reset_password': True, 'mail_create_nolog': True}).create({
+        user = cls.env['res.users'].with_context({'mail_create_nolog': True}).create({
```

Comment:
Not sure, just find it odd given the rest of the diff
```suggestion
        user = cls.env['res.users'].with_context(mail_create_nolog=True).create({
```

Reply:
That's correct indeed, those 3 seems to be the last occurrences of users being created (in tests) with a context being fully overridden (at least by quickly grepping).